packages/mdc-top-app-bar/README.md

```diff
@@ -51,6 +51,18 @@ npm install @material/top-app-bar
 </header>
 ```
 
+#### Menu Icon
```

Comment:
Nit: Menu Icon -> Icons, since this is also applicable to action items discussed under variants.